python/mlc_chat/quantization/quantization.py

```diff
@@ -200,6 +203,7 @@ def quantize_weight(self, weight: tvm.runtime.NDArray) -> List[tvm.runtime.NDArr
         storage_dtype="uint32",
         model_dtype="float16",
         quantize_embedding=False,
+        quantize_linear=False,
     ),
     "fp16_max_calibration": PerTensorQuantize(
```

Comment:
Do we need this quantization method for anything other than debugging?

Reply:
I think this is what we should go to prod with as it has the best quality and the linear layers that are left over after MOE replacement are all small and not a significant factor in compute.